gapic/schema/api.py

```diff
@@ -74,6 +81,36 @@ def module_name(self) -> str:
         """
         return to_snake_case(self.name.split('/')[-1][:-len('.proto')])
 
+    @cached_property
+    def python_modules(self) -> Tuple[Sequence[str], str]:
```

Comment:
Sorry, I think I meant that shouldn't this be `Sequence[Tuple[str, str]]` like here: https://github.com/googleapis/gapic-generator-python/blob/9851ddb75b4b7bd26fd3fb1fa2ac0ba5c8054185/gapic/schema/wrappers.py#L440

Reply:
You are correct. Fixed.